pulp_python/app/viewsets.py

```diff
@@ -82,6 +93,15 @@ def create(self, request):
         return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
 
 
+class PythonRemoteFilter(filterset.FilterSet):
+    class Meta:
+        model = python_models.PythonRemote
+        fields = {
+            'name': ['exact', 'in'],
+            'last_updated': ['lt', 'lte', 'gt', 'gte', 'range'],
+            'projects': ['contains']
```

Comment:
I changed this to a 'contains' filter instead of an ['exact', 'in'] filter because that make a lot more sense for a string of projects... It might make sense to add ['in'] if we make this field into a json.